sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-2021-2022-awards-amount-CSGD.e2e-spec.ts

```diff
@@ -222,5 +232,15 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
           .limitAwardCSGDAmount,
       ),
     );
+    expect(calculatedAssessment.variables.federalAwardNetCSGDAmount).toBe(
+      Math.min(
+        calculatedAssessment.variables.calculatedDataTotalRemainingNeed2,
+        calculatedAssessment.variables.federalAwardCSGDAmount -
+          assessmentConsolidatedData.programYearTotalPartTimeCSGD,
+      ),
+    );
+    expect(
+      calculatedAssessment.variables.finalFederalAwardNetCSGDAmount,
+    ).toEqual(calculatedAssessment.variables.federalAwardNetCSGDAmount);
```

Comment:
I will say that we need to add one more test by injecting a `programYearTotalPartTimeCSGD` and observing the award calculated.